src/data/data.js

```diff
@@ -1,5 +1,10 @@
 //>>excludeStart("exclude", pragmas.exclude);
-define([ "shoestring" ], function(){
+define([
+	"shoestring",
+//>>includeStart("development", pragmas.development);
+	"dom/is"
+//>>includeEnd("development");
```

Comment:
I've never done a pragma inside another pragma. This probably depends on the order in which they are run? Not sure what to do here.